src/ProxyClient/Varnish.php

```diff
@@ -125,28 +125,36 @@ public function refresh($url, array $headers = array())
 
         return $this;
     }
-
-    /**
-     * {@inheritdoc}
-     *
-     * @throws MissingHostException If a relative path is queued for purge/
-     *                              refresh and no base URL is set
-     *
-     */
-    protected function createRequest($method, $url, array $headers = array())
+    
+    protected function queueRequest($method, $url, array $headers = array())
     {
-        $request = parent::createRequest($method, $url, $headers);
-
-        // For purge and refresh, add a host header to the request if it hasn't
-        // been set
-        if (self::HTTP_METHOD_BAN !== $method
-            && '' == $request->getHeader('Host')
-        ) {
-            throw MissingHostException::missingHost($url);
+        if (self::HTTP_METHOD_BAN !== $method) {
+            // AND NO HOST IS SET
```

Comment:
The `MissingHostException` makes sense: *usually* you don’t want to accidentally send purge/refresh requests to `127.0.0.1` if your app runs on `my-app.com` (because the hostnames is included in the hash, so the purged `127.0.0.1/purge-this` does not match `my-app.com/127.0.0.1`). 

But what if I decide to run my app on an IP address? In that case, I would want to be able to send purges/refreshes to `127.0.0.1/purge-this`. Should we always throw this exception?